image/image_test.go

```diff
@@ -190,6 +192,8 @@ func (s *imageSuite) SnapAction(_ context.Context, _ []*store.CurrentSnap, actio
 	if _, instanceKey := snap.SplitInstanceName(actions[0].InstanceName); instanceKey != "" {
 		return nil, fmt.Errorf("unexpected instance key in %q", actions[0].InstanceName)
 	}
+	// record
+	s.storeActions = append(s.storeActions, actions[0])
```

Comment:
Code above checks that `actions[0]` exits. In case a fellow reviewer was wondering.